frontend/src/store.js

```diff
@@ -0,0 +1,29 @@
+import Vue from 'vue'
+import Vuex from 'vuex'
+
+Vue.use(Vuex)
```

Comment:
You can take this out since it's in the other file now.

Reply:
Hi Thomas, after removing it, there was a reference error in the browser console:

![image](https://user-images.githubusercontent.com/68148934/98459992-0cb9f500-21db-11eb-8cca-94d66dcf2269.png)

Reply:
Oh, weird. You didn't remove the import statement right? You still need that here.

Reply:
Oh you are right, i removed the import statement too, my bad! 

have tried again with just the `Vue.use(Vuex)` removed. The frontend page does not load and the error on the console is:
![image](https://user-images.githubusercontent.com/68148934/98461434-0893d480-21e7-11eb-9858-dde83d55f5c7.png)

Reply:
Ok, can leave it then.

Reply:
Okay, have removed it from the other file :)